jazz_services-handler/index.js

```diff
@@ -123,16 +125,26 @@ var processRecord = function (record, configData,authToken) {
 			});
 	});
 }
-var checkInterest = function (encodedPayload, sequenceNumber) {
+var checkInterest = function (encodedPayload, sequenceNumber, configData) {
 	return new Promise((resolve, reject) => {
-		var payload = JSON.parse(new Buffer(encodedPayload, 'base64').toString('ascii'));
-		if (payload.Item.EVENT_TYPE && payload.Item.EVENT_TYPE.S) {
-			logger.info("found " + payload.Item.EVENT_TYPE.S + " event with sequence number: " + sequenceNumber);
-			resolve({
-				"interested_event": true,
-				"payload": payload.Item
-			});
-		} 
+		var kinesisPayload = JSON.parse(new Buffer(encodedPayload, 'base64').toString('ascii'));
+		logger.info("kinesisPayload : " + JSON.stringify(kinesisPayload));
+		if (kinesisPayload.Item.EVENT_TYPE && kinesisPayload.Item.EVENT_TYPE.S) {
+			if (_.includes(configData.EVENTS.EVENT_TYPE, kinesisPayload.Item.EVENT_TYPE.S) &&
```

Comment:
you can avoid using includes here and use ES5 native includes. Also, other places